code/datums/status_effects/debuffs.dm

```diff
@@ -913,6 +913,14 @@
 	examine_text = "<span class='warning'>SUBJECTPRONOUN has a blank, catatonic like stare.</span>"
 	alert_type = /atom/movable/screen/alert/status_effect/ghoul
 
+/datum/status_effect/ghoul/get_examine_text()
+	var/mob/living/carbon/human/H = owner
```

Comment:
This variable is probably unecessary, just use owner

Reply:
Using owner doesn't even let me compile it because it sees "owner" as mob/living, I used the variable to explicitly make it human